YesterdayApi/Core/Email/IEmailReceiver.cs

```diff
@@ -0,0 +1,8 @@
+namespace YesterdayApi.Core.Email
+{
+    public interface IEmailReceiver
```

Comment:
Another abstraction which we don't need. I was looking for UserName inside of User.

Reply:
UserName is inherited from the AspIdentityUser class. As the SOLID interface segregation principle says: “Clients should not be forced to depend upon interfaces that they do not use.”, thats why i created **IEmailReceiver** that has everything what the service needs

Reply:
I think this is actually valid, as it adheres to Interface Segregation Principle, as email service doesn't need to know about the hashed password or confirmedPhone, etc.

Reply:
Yeah. Good catch